facdb/sql/dycd_afterschoolprograms.sql

```diff
@@ -0,0 +1,41 @@
+DROP TABLE IF EXISTS _dycd_afterschoolprograms;
+
+SELECT
+    uid,
+    source,
+    site_name as facname,
+    parsed_hnum as addressnum,
+    parsed_sname as streetname,
+    location_1 as address,
+    NULL as city,
+    postcode as zipcode,
+    SPLIT_PART(borough__community, ',', 1) as boro,
+    NULL as borocode,
+    bin,
+    bbl,
+    REPLACE(program, 'NDA Immigrats', 'NDA Immigrants')||' '||program_type as factype,
```

Comment:
Update to `REPLACE(program, 'NDA Immigrats', 'NDA Immigrants')||': '||program_type as factype`